nix/m1-support/m1n1/default.nix

```diff
@@ -70,7 +70,6 @@ in stdenv.mkDerivation rec {
     runHook preInstall
 
     mkdir -p $out/build
-    cp build/m1n1.macho $out/build
```

Comment:
Is the `.macho` still useful for the hypervisor and remote download functionality? Doesn't seem harmful to leave it and obviously m1n1 still builds it.

Reply:
It appears the hypervisor switched over from requiring macho to supporting raw images in https://github.com/AsahiLinux/m1n1/commit/0cf1f393ed1409a0bb80696d3984b03a53db66cf

Reply:
Okay sounds good, it's easy to put it back if necessary later.